packages/terra-list/src/List.scss

```diff
@@ -50,14 +51,20 @@
     }
   }
 
+
   .is-selectable {
     cursor: pointer;
 
-    &.list-item:hover,
-    &.list-item:focus {
+    &.list-item:hover {
       background-color: var(--terra-list-item-hover-background-color, lighten(#ebf6fd, 2%)) !important;
     }
 
+    &.list-item:focus {
+      background-color: var(--terra-list-item-focus-background-color, lighten(#ebf6fd, 2%)) !important;
+      outline: var(--terra-list-item-focus-outline);
+      box-shadow: var(--terra-list-item-focus-box-shadow);
```

Comment:
Should have a fallback value for this and others too?

Reply:
No fallback value is needed if the default theme does not need these styles.